custom_components/hacs/base.py

```diff
@@ -876,6 +878,11 @@ async def async_get_category_repositories_experimental(self, category: str) -> N
                 continue
             if repo in self.common.archived_repositories:
                 continue
+            try:
+                V2_REPO_SCHEMA[category](repo_data)
+            except vol.Invalid as exception:
+                self.log.info("Got invalid data for %s (%s)", repo, exception)
+                continue
```

Comment:
This could happen at the beginning of the for loop instead, since we make assumptions about the data there:
```
repo = repo_data["full_name"]
```

Reply:
True, but the other removed/archived checks that happen are cheaper than this schema validation right?
So if there are some we can skip that is great?

Reply:
Yeah, OK